### PR TITLE
Pgr migration

### PR DESCRIPTION
**Pull request by shashwat-egov**

### CODE REVIEW

File: rainmaker-pgr/src/main/java/org/egov/pgr/contract/ServiceReqSearchCriteria.java [hunk 1]
```diff
@@ -70,8 +70,8 @@
 	@JsonProperty("createdBy")
 	private String createdBy;
 	
-	@JsonProperty("noOfRecords")
-	private Long noOfRecords;
+	@JsonProperty("limit")
```
Comment (talele08) [inline rainmaker-pgr/src/main/java/org/egov/pgr/contract/ServiceReqSearchCriteria.java:73]: Is this change required? Now that we are not using searcher

**Review by talele08 — CHANGES_REQUESTED** [review]

File: rainmaker-pgr/src/main/java/org/egov/pgr/contract/ServiceReqSearchCriteria.java (hunk 1 above)
Comment (shashwat-egov) [reply rainmaker-pgr/src/main/java/org/egov/pgr/contract/ServiceReqSearchCriteria.java:73]: Not needed anymore.

File: rainmaker-pgr/src/main/java/org/egov/pgr/service/GrievanceService.java [hunk 2]
```diff
@@ -637,7 +637,7 @@ public ServiceResponse prepareResult(Object response, RequestInfo requestInfo) {
 				services.add(obj.getServices());
 			}
 		});
-		replaceIdsWithUrls(actionHistory);
+		//replaceIdsWithUrls(actionHistory);
```
Comment (talele08) [inline rainmaker-pgr/src/main/java/org/egov/pgr/service/GrievanceService.java:640]: make this configurable

**Review by talele08 — CHANGES_REQUESTED** [review]

File: rainmaker-pgr/src/main/resources/application.properties [hunk 3]
```diff
@@ -163,3 +163,8 @@ egov.default.expiry.time.before.reopen.in.hours=120
 
 logging.level.org.egov.tracer=debug
 
+#Migration
+kafka.migration.topic=pgr-migration
+migration.batch.size=100
+
+url.enrichment.enabled=false
```
Comment (talele08) [inline rainmaker-pgr/src/main/resources/application.properties:170]: make it true as default

**Review by talele08 — CHANGES_REQUESTED** [review]

**Review by talele08 — APPROVED** [review]
